AWBWApp.Game/API/Replay/ReplayBuilding.cs

```diff
@@ -37,7 +37,8 @@ public bool DoesBuildingMatch(ReplayBuilding other)
                 return false;
             if (Position != other.Position)
                 return false;
-            if (Capture != other.Capture)
+            //For destroyed Pipeseams Capture is undefined
+            if (TerrainID != 115 && TerrainID != 116 && Capture != other.Capture)
```

Comment:
Might be best if these numbers were named constants like `pipe_seam` and `pipe_seam_destroyed`